scripts/categories/sparql/bl-categories.sparql

```diff
@@ -0,0 +1,25 @@
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX meta: <https://w3id.org/biolink/biolinkml/meta/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX BFO: <http://purl.obolibrary.org/obo/BFO_>
+PREFIX biolink: <https://w3id.org/biolink/vocab/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX : <..>
+
+
+CONSTRUCT {
+  ?type biolink:category ?category .
+}
+WHERE {
+  {
+    GRAPH :biolink-model.ttl  { ?category skos:mappingRelation ?class . }
+    GRAPH :monarch-slim.owl  {
+      ?type (a|rdfs:subClassOf|owl:equivalentClass|owl:sameAs|BFO:0000050)* ?class .
```

Comment:
You are making the assumption that equiv/sameAs symmetry entailments are asserted

(this is part of a general issue, see https://github.com/TranslatorSRI/reference-kg/issues/9)

I think you also need to account for assertion in the opposite direction with `^`